page-optimize.php

```diff
@@ -265,38 +269,65 @@ function page_optimize_schedule_cache_cleanup() {
 }
 
 // Cases when we don't want to concat
-function page_optimize_bail() {
+function page_optimize_should_optimize() {
```

Comment:
Since I wanted to publish a related filter for this function, I was looking for a better, non-slang word for "bail", but none satisfied me. I ended up thinking it was better to remove the negation and make this a positive "should optimize" along with a filter of the same name.